internal/test/wmcb/wmcb_test.go

```diff
@@ -297,6 +328,48 @@ func (vm *wmcbVM) runTestKubeletUninstall(t *testing.T) {
 	require.NoError(t, err, "TestKubeletUninstall failed")
 }
 
+// configureContainerd installs the containerd Windows service
+func (vm *wmcbVM) configureContainerd() error {
+	// Check if containerd is running
+	out, err := vm.Run("Get-Process -Name \"containerd\"", true)
+
+	// stderr being empty implies that containerd was running.
+	if err == nil || out == "" {
+		return nil
+	}
+
+	out, err = vm.Run(mkdirCmd(kLog), false)
+	if err != nil {
+		return fmt.Errorf("unable to create remote directory %s: %v\n%s", kLog, err, out)
+	}
+
+	// set Windows defender exclusions for containerd
+	setExclusionCmd := fmt.Sprintf("Add-MpPreference -ExclusionProcess %s", containerdPath)
+	out, err = vm.Run(setExclusionCmd, true)
+	if err != nil {
+		return fmt.Errorf("setting Windows defender process exclusion failed: %v", err)
+	}
+
+	// Start the containerd Windows service
+	binPath := fmt.Sprintf("%s --config %s --log-file %s\\containerd.log --log-level=info --run-service",
```

Comment:
Considering this is an in-development feature, should we start with `--log-level=debug` or will be too much?